Scripts/Runtime/Entities/Lifecycle/Spawner/EntitySpawner.cs

```diff
@@ -157,16 +177,40 @@ public Entity SpawnWithPrototypeImmediate(TEntitySpawnDefinition spawnDefinition
         // SPAWN API - IN JOB
         //*************************************************************************************************************
 
-        public JobHandle AcquireEntitySpawnWriterAsync(out EntitySpawnWriter<TEntitySpawnDefinition> entitySpawnWriter)
+        public AccessControlledValue<EntitySpawnWriter<TEntitySpawnDefinition>>.AccessHandle AcquireWithSharedWriteHandle()
         {
+            DispatchPendingWorkAdded();
+
+            var handle = m_DefinitionsToSpawn.AcquireWithHandle(AccessType.SharedWrite);
+            EntitySpawnWriter<TEntitySpawnDefinition> writer = new EntitySpawnWriter<TEntitySpawnDefinition>(handle.Value.AsWriter());
+            return AccessControlledValue<EntitySpawnWriter<TEntitySpawnDefinition>>.AccessHandle.CreateDerived(handle, writer);
+        }
+
+        public EntitySpawnWriter<TEntitySpawnDefinition> AcquireSharedWrite()
+        {
+            DispatchPendingWorkAdded();
+
+            UnsafeTypedStream<SpawnDefinitionWrapper<TEntitySpawnDefinition>> definitionsToSpawn = m_DefinitionsToSpawn.Acquire(AccessType.SharedWrite);
+            return new EntitySpawnWriter<TEntitySpawnDefinition>(definitionsToSpawn.AsWriter());
+        }
+
+        public JobHandle AcquireSharedWriteAsync(out EntitySpawnWriter<TEntitySpawnDefinition> value)
```

Comment:
```suggestion
        public JobHandle AcquireSharedWriteAsync(out EntitySpawnWriter<TEntitySpawnDefinition> entitySpawnWriter)
```

Reply:
Is that the style? To change to a context specific parameter name when implementing an interface? I thought we'd want to match the interface.

Reply:
Hmmmn right... good point. 

Disregard.